File: generator/src/com/company1/BankCard.java
package com.company1;

public class BankCard {

    public Long id;
    public String digitCard;
    public int cvv;
    public String actionTime;
    public String cardHolder;

    @Override
    public String toString() {
        return String.format("Id = %o, digitCard = %s, cvv =%o,actionTime = %s,  cardHolder = %s",
                this.id, this.digitCard, this.cvv, this.actionTime, this.cardHolder);
    }
}

File: shop-app/src/app/models/bank-card.ts
export class BankCard {
  id: number | undefined;
  digitCard: number | undefined;
  cvv: number | undefined;
  actionTime: string | undefined;
  cardHolder: string | undefined;


  constructor(digitCard: number, cvv: number, actionTime: string, cardHolder: string) {
    this.digitCard = digitCard;
    this.cvv = cvv;
    this.actionTime = actionTime;
    this.cardHolder = cardHolder;
  }
}
